doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md

```diff
@@ -5,4 +5,162 @@ permalink: /:path/
+1. Install CC3200SDK.
+    1. Install [Wine](https://www.winehq.org/).
+
+            sudo apt-get install wine
+    1. [Download](http://www.ti.com/tool/cc3200sdk) CC3200 SDK.
+    1. Unpack CC3200 SDK.
+    1. Change the configuration file for the debug interface.
+        In `/opt/kaa/cc3200-sdk/tools/gcc_scripts/cc3200.cfg` replace the following few lines:
+
+            interface ft2232
+            ft2232_layout luminary_icdi
+            ft2232_device_desc "USB <-> JTAG/SWD"
+            ft2232_vid_pid 0x0451 0xc32a
+
+        with
+
+            interface ftdi
+            ftdi_device_desc "USB <-> JTAG/SWD"
+            ftdi_vid_pid 0x0451 0xc32a
+            ftdi_layout_init 0x00a8 0x00eb
+            ftdi_layout_signal nSRST -noe 0x0020
+    1. Edit GDB configuration.
+
+            sed -i 's#-f cc3200.cfg#-f /opt/kaa/cc3200-sdk/tools/gcc_scripts/cc3200.cfg#g' /opt/kaa/cc3200-sdk/tools/gcc_scripts/gdbinit
+
+
+# Installing necessary components for Windows
+
+Before building the C SDK for the CC3200 platform on Windows, you need to perform
+the following installation.
+
+1. Install [Cygwin](https://www.cygwin.com/) with the additional following packages:
+    * `Archive/unzip`
+    * `Archive/zip`
+    * `Devel/autoconf`
+    * `Devel/automake`
+    * `Devel/libtool`
+    * `Devel/subversion` (Note: if using TortoiseSVN/Windows7, skip this file)
+    * `Devel/make`
+    * `Devel/gcc-core`
+    * `Devel/gcc-g++`
+    * `Devel/mingw-gcc-core`
+    * `Devel/mingw-gcc-g++`
+    * `Devel/mingw-runtime`
+
+
+    Refer to [SO question page](http://superuser.com/questions/304541/how-to-install-new-packages-on-cygwin) for details how to install packages on Cygwin.
+1. Install the GNU ARM toolchain: [gcc-arm-none-eabi](https://launchpad.net/gcc-arm-embedded)
+to the `opt\kaa` directory under the Cygwin root directory (default is `C:\cygwin`)
+1. Install [CC3200SDK](http://www.ti.com/tool/cc3200sdk) to the `opt\kaa`
+(if the directory doesn't exist, create it) directory under the Cygwin root directory.
+1. Install [cmake](http://www.cmake.org/) and add it bin directory to the system environment.
+
+To enable debugging for your CC3200 applications, you will also need to build OpenOCD as
+described in [the official CC3200-Getting_Started_Guide][cc3200-getting-started-guide] (item 3.3.3)
+
+For more information, please refer to the official [CC3200-Getting_Started_Guide][cc3200-getting-started-guide]
+
+[cc3200-getting-started-guide]: http://www.ti.com/lit/ug/swru376d/swru376d.pdf
+
+
+# Creating applications based on C SDK
+
+## Building C SDK for Linux
+
+Before creating applications based on the C SDK, you should obtain the C SDK and build a static
+library from it. To do so, generate the [C SDK in Admin UI]({{root_url}}/Administration-guide/Tenants-and-applications-management/#generating-endpoint-sdk),
+then extract the archive, cd to it and execute the following for Linux:
```

Comment:
Some changes were made here in another pull request. Currently we have: *Change directory to where SDK was unpacked and execute the following for:*
**for** is not needed.